static/app/utils/replays/hooks/useReplayData.spec.tsx

```diff
@@ -428,7 +428,8 @@ describe('useReplayData', () => {
       expect(result.current).toStrictEqual(
         expect.objectContaining({
           attachments: mockSegmentResponse,
-          errors: mockErrorResponse,
+          // mockErrorResponse is the same between both responses
+          errors: [...mockErrorResponse, ...mockErrorResponse],
           replayRecord: expectedReplay,
```

Comment:
nice. the test could use some more variety in the mocks. but this seems to work!